packages/webapp/src/components/Form/CompositionInputs/NumberInputWithSelect.tsx

```diff
@@ -24,55 +24,47 @@ import styles from './styles.module.scss';
 
 export type Option = { value: string; label: ReactNode };
 
-export const UNIT = 'npk_unit';
-
-export enum NPK {
-  N = 'n',
-  P = 'p',
-  K = 'k',
-}
-
 export enum Unit {
```

Comment:
Do we still need this here? Can Units be anything now?

Reply:
I changed the type to `unit: Unit | string;`, so units can be anything, but I guess it's weird to use the enum as const...😅  I'll change the enum to const!